services/web/server/tests/unit/with_dbs/04/test_catalog_api.py

```diff
@@ -50,37 +49,33 @@ def client(loop, app_cfg, aiohttp_client, postgres_db):
 
 
 @pytest.fixture
-def mock_api_client_session(client, mocker):
-    get_client_session = mocker.patch(
-        "simcore_service_webserver.catalog_client.get_client_session"
-    )
-
-    class MockClientSession(mocker.MagicMock):
-        async def __aenter__(self):
-            # Mocks aiohttp.ClientResponse
-            # https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse
-            resp = mocker.Mock()
+def patch_app_client_session(client, mocker):
+    async def create_200_empty_response(*args, **kwargs):
+        # Mocks aiohttp.ClientResponse
+        # https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse
+        resp = mocker.MagicMock(name="SuccessfulEmptyResponse")
```

Comment:
note: probably aioresponses would be more readable for these kinds of mocks.

Reply:
yep. I created a new fixture in ``pytest_simcore`` called ``aioresponses_mocker`` and replace this as well